src/utils/barn.ts

```diff
@@ -139,3 +150,42 @@ export const oppdaterBarnLabels = (barn: IBarn[], intl: LokalIntlShape) => {
 
   return oppdaterteBarn;
 };
+
+const unikeForelderidenter = (barn: IBarn[]) =>
+  Array.from(new Set(barn.map((b) => b.forelder?.ident?.verdi))).filter(
+    (forelderIdent): forelderIdent is string => forelderIdent !== undefined
+  );
+
+export const forelderidentMedBarn = (barn: IBarn[]) =>
+  new Map(
+    unikeForelderidenter(barn).map((forelderIdent) => [
+      forelderIdent,
+      barn.filter((barn) => barn.forelder?.ident?.verdi === forelderIdent),
+    ])
+  );
```

Comment:
Vil alle annenForelder som er registrert i pdl ha ident?

Reply:
Ja, det er en forutsetning for å "hente" annen forelder fra pdl. Vi får tilbake en ident, og henter ut navn+++ basert på denne.